### ISSUE TITLE
GraphQL Spring Security example code

### ISSUE THREAD

**Issue by for3v3rInFriendZone** [g00glen00b/gatsby-blog#72]
**Blogpost**:

I have a question about [Title of blogpost](https://dimitr.im/link-to-blogpost).

**Description**:

Hi Dimitri,

can you provide me a link to your github example from this blogpost:
https://dimitr.im/graphql-spring-security

Thanks.

**Comment by devAjayAj** [comment]
Looking for the same, can you please provide Github repo link for the above mentioned article.

**Comment by g00glen00b** [comment]
@for3v3rInFriendZone and @devAjayAj, I just published the project that I wrote to write tutorial. It can be found here: https://github.com/g00glen00b/whoiswho-graphql 

Be careful, it's a full project including many other features, but it does demonstrate the security-part and how I combined it with React. 

A link to the project will also be included into the tutorials (it's being deployed as we speak).

I hope it helps.
